docs/io/output/vpacket_logging.rst

```diff
@@ -0,0 +1,33 @@
+**********************
+Virtual Packet Logging
+**********************
+
+If ``virtual_packet_logging`` is set to ``True`` in either the :ref:`spectrum configuration <spectrum-config>` or as 
+an :ref:`argument to the run_tardis function <optional-input>`, then TARDIS will log information about the
```

Comment:
Might also want to elaborate a bit on `<optional-input>` page how to use this argument by an e.g. (I don't know who added that but they should have added a code e.g. atleast)